tests/scalers/azure/azure_aad_pod_identity/azure_aad_pod_identityte_test.go

```diff
@@ -0,0 +1,152 @@
+//go:build e2e
```

Comment:
I guess that this test'd be better inside `internals` folder because we already have there another test case for ScaledObject validations. I'd rename this file to something like `trigger_authentication_validation` too (and also namespace and those stuff)